src/admin-views/aggregator/settings.php

```diff
@@ -673,7 +675,7 @@
 				'Use the options below to configure your imports. Global Import Settings apply to all imports, but you can also override the global settings by adjusting the origin-specific options. Check your Event Aggregator Service Status on the %1$s.',
 				'the-events-calendar'
 			),
-			'<a href="' . Tribe__Settings::instance()->get_url( array( 'page' => 'tribe-help' ) ) . '#tribe-tribe-aggregator-status">' . esc_html__( 'Help page', 'the-events-calendar' ) . '</a>'
+			'<a href="' . Tribe__Settings::instance()->get_url( [ 'page' => 'tribe-help' ] ) . '#tribe-tribe-aggregator-status">' . esc_html__( 'Help page', 'the-events-calendar' ) . '</a>'
```

Comment:
:warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 23f3f196672682dd25591370e62cd579</sub>